build/yamls/base/conf/antrea-controller.conf

```diff
@@ -27,3 +27,8 @@ featureGates:
 # And the Secret must be mounted to directory "/var/run/antrea/antrea-controller-tls" of the
 # antrea-controller container.
 #selfSignedCert: true
+
+# Enable custom admission controller webhooks to mutate/validate resources. For example, it
+# enables the labels mutator which labels all Namespaces with the reserved
+# "antrea.io/metadata.name = <namespaceName>" label.
+#enableCustomAdmissionControllers: false
```

Comment:
I know you want to use a generic option for future extension, but a name like enableNamespaceLabelMutator might be easier to understand?
Probably you already have plans to add new admission controllers later? But do you think we might like to enable/disable them individually?

Reply:
after your comment, initially i was thinking of maybe introduce a new option enableCustomAdmissionControllers as a list of strings, which includes values like "namespaceLabelMutator" and newer values in the future etc. but maybe we won't have that many controllers so individual flags may be okay. will update according to your suggestion and let me know if you think other approach is better